game/src/debug/routes.rs

```diff
@@ -208,28 +208,40 @@ fn params_to_controls(ctx: &mut EventCtx, mode: TripMode, params: &RoutingParams
                 .margin_right(20),
             Spinner::widget(
                 ctx,
+                "unprotected turn penalty",
                 (1, 100),
                 params.unprotected_turn_penalty.inner_seconds() as isize,
-            )
-            .named("unprotected turn penalty"),
+            ),
         ]));
     }
     if mode == TripMode::Bike {
         // TODO Spinners that natively understand a floating point range with a given precision
         rows.push(Widget::row(vec![
             "Bike lane penalty:".text_widget(ctx).margin_right(20),
-            Spinner::widget(ctx, (0, 20), (params.bike_lane_penalty * 10.0) as isize)
-                .named("bike lane penalty"),
+            Spinner::widget(
+                ctx,
+                "bike lane penalty",
+                (0, 20),
+                (params.bike_lane_penalty * 10.0) as isize,
```

Comment:
Other widgetry refactors on my mind: making Spinner generic, so we can directly hand it Distance, Duration, f64, etc and a step size.